src/Databases/PostgreSQL/DatabasePostgreSQL.cpp

```diff
@@ -36,6 +36,7 @@ namespace ErrorCodes
 
 static const auto suffix = ".removed";
 static const auto cleaner_reschedule_ms = 60000;
+static const auto reschedule_error_multiplier = 10;
```

Comment:
let's just update `cleaner_reschedule_ms` to a bigger value

Reply:
I think differentiate the normal case and unusual case is more reasonable. reschedule_error_multiplier is only used in the abnormal case.

Reply:
ok indeed